website/docs/r/mq_broker.html.markdown

```diff
@@ -56,18 +56,18 @@ The following arguments are supported:
   are applied immediately, or during the next maintenance window. Default is `false`.
 * `auto_minor_version_upgrade` - (Optional) Enables automatic upgrades to new minor versions for brokers, as Apache releases the versions.
 * `broker_name` - (Required) The name of the broker.
-* `configuration` - (Optional) Configuration of the broker. See below.
+* `configuration` - (Optional) Configuration of the broker. See below. (Applies to `ActiveMQ` only)
 * `deployment_mode` - (Optional) The deployment mode of the broker. Supported: `SINGLE_INSTANCE` and `ACTIVE_STANDBY_MULTI_AZ`. Defaults to `SINGLE_INSTANCE`.
 * `encryption_options` - (Optional) Configuration block containing encryption options. See below.
-* `engine_type` - (Required) The type of broker engine. Currently, Amazon MQ supports only `ActiveMQ`.
+* `engine_type` - (Required) The type of broker engine. Currently, Amazon MQ supports `ActiveMQ` and `RabbitMQ`.
 * `engine_version` - (Required) The version of the broker engine. See the [AmazonMQ Broker Engine docs](https://docs.aws.amazon.com/amazon-mq/latest/developer-guide/broker-engine.html) for supported versions.
 * `host_instance_type` - (Required) The broker's instance type. e.g. `mq.t2.micro` or `mq.m4.large`
 * `publicly_accessible` - (Optional) Whether to enable connections from applications outside of the VPC that hosts the broker's subnets.
 * `security_groups` - (Required) The list of security group IDs assigned to the broker.
 * `subnet_ids` - (Optional) The list of subnet IDs in which to launch the broker. A `SINGLE_INSTANCE` deployment requires one subnet. An `ACTIVE_STANDBY_MULTI_AZ` deployment requires two subnets.
 * `maintenance_window_start_time` - (Optional) Maintenance window start time. See below.
-* `logs` - (Optional) Logging configuration of the broker. See below.
-* `user` - (Required) The list of all ActiveMQ usernames for the specified broker. See below.
+* `logs` - (Optional) Logging configuration of the broker. See below. (Applies to `ActiveMQ` only)
+* `user` - (Required) The list of all broker usernames for the specified broker. See below. RabbitMQ users can only be specified in the creation of the resource. Updates to users can only be in the RabbitMQ UI.
```

Comment:
It might be helpful to add that RabbitMQ comes with [the management plugin enabled](https://docs.aws.amazon.com/amazon-mq/latest/developer-guide/rabbitmq-basic-elements-plugins.html), so it's also possible to the API to created/update/delete users (and more, `/api/definitions` endpoint is super handy  for that).

We're using the [cloud formation workaround](https://github.com/hashicorp/terraform-provider-aws/issues/16030#issuecomment-778554380) at the moment, and also have a Lambda in place (triggered by Terraform) to set up a couple of exchanges and users.

Reply:
Unrelated but you can use the rabbit provider to create exchanges and users :D

Reply:
Ah yes, assuming that where the place where you run `terraform apply` has direct access to the RMQ server.